app/src/main/java/com/example/househomey/HomeFragment.java

```diff
@@ -133,4 +133,30 @@ private void showFilterMenu(View view) {
         popupMenu.show();
     }
 
-}
+    /**
+     * Called when a filter is applied or modified.
+     * @param filterType  The type of filter being applied (e.g., "MAKE").
+     * @param filterValue The value or criteria for the filter.
+     */
+    @Override
+    public void onFilterApplied(String filterType, String filterValue) {
```

Comment:
I would prefer to see this split into two parts, ```setFilter(filterValue)``` and ```applyFilter()```. My thinking is this, our app theoretically live updates from the database, that means every time we update in the db we should apply the current filter. So if we split these and then call applyfilter() at the end of the snapshot listener. Doesn't have to be that exact idea if you have a better way of filtering out updates from the db.

Reply:
Further to that point, if we want filters to be saved between fragment navs (idk if we want that) they would disappear.